docs/serving/samples/README.md

```diff
@@ -4,7 +4,7 @@ Serving resources and how they can be applied across common use cases.
 
 | Name                       | Description                                                                                                                                                                                                              |                                                                                                                                                                                                                            Languages                                                                                                                                                                                                                             |
 | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-| Hello World                | A quick introduction that highlights how to deploy an app using Knative Serving.                                                                                                                                         | [C#](./hello-world/helloworld-csharp/README.md), [Go](./hello-world/helloworld-go/README.md), [Java](./hello-world/helloworld-java/README.md), [Kotlin](./hello-world/helloworld-kotlin/README.md), [Node.js](./hello-world/helloworld-nodejs/README.md), [PHP](./hello-world/helloworld-php/README.md), [Python](./hello-world/helloworld-python/README.md), [Ruby](./hello-world/helloworld-ruby/README.md), [Scala](./hello-world/helloworld-scala/README.md) |
+| Hello World                | A quick introduction that highlights how to deploy an app using Knative Serving. The Knative Serving [Community Docs](../../../community/samples/serving) has Clojure, Dart, Elixir, Haskell, Rust, Shell, Swift and Vertx.                                                                                                                                         | [C#](./hello-world/helloworld-csharp/README.md), [Go](./hello-world/helloworld-go/README.md), [Java](./hello-world/helloworld-java/README.md), [Kotlin](./hello-world/helloworld-kotlin/README.md), [Node.js](./hello-world/helloworld-nodejs/README.md), [PHP](./hello-world/helloworld-php/README.md), [Python](./hello-world/helloworld-python/README.md), [Ruby](./hello-world/helloworld-ruby/README.md), [Scala](./hello-world/helloworld-scala/README.md) |
```

Comment:
Instead of adding the link to the community samples in the table, I'd recommend linking to it above.

Suggested text: "Additional samples provided by the community are available in the Community Samples folder."

No need to spell out which languages are in the community folder, as that will be difficult to keep accurate over time.

Reply:
Change addressed, Thanks for your review!